inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/SinkContext.java

```diff
@@ -114,16 +120,16 @@ public void reload() {
 
     /**
      * get clusterId
-     *
+     * 
```

Comment:
Please do not add a space at the end of the line.

Reply:
fixed